src/app-account.js

```
@@ -56,17 +65,27 @@
     }
+    case 'openid': {
+      let { error, url } = await loginWithOpenIdSetup(req.body);
+      if (error) {
+        res.status(400).send({ status: 'error', reason: error });
+        return;
+      }
+      res.send({ status: 'ok', data: { redirect_url: url } });
+      return;
+    }
+
     case 'password':
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)